script/fork-helpers/SimulateVote.s.sol

```diff
@@ -144,9 +145,71 @@ contract SimulateVote is Script, DeploymentFixtures, ForkHelpersCommon {
         accounting = CSAccounting(deploymentConfig.accounting);
         oracle = CSFeeOracle(deploymentConfig.oracle);
         IBurner burner = IBurner(locator.burner());
+        CSParametersRegistry parametersRegistry = CSParametersRegistry(
+            deploymentConfig.parametersRegistry
+        );
 
         vm.startBroadcast(admin);
 
+        uint256 identifiedCommunityStakerCurveId = 1;
```

Comment:
This block should be in deploy, not in the vote since we can set it all before the vote